jfr-daemon/src/main/java/com/newrelic/jfr/daemon/SetupUtils.java

```diff
@@ -50,10 +50,14 @@ public static Attributes buildCommonAttributes(DaemonConfig config) {
             .put(AttributeNames.INSTRUMENTATION_PROVIDER, "JFR-Uploader")
             .put(AttributeNames.COLLECTOR_NAME, "JFR-Uploader");
     String hostname;
-    try {
-      hostname = InetAddress.getLocalHost().toString();
-    } catch (Throwable e) {
-      hostname = InetAddress.getLoopbackAddress().toString();
+    if (config.getHostname() == null) {
```

Comment:
Not sure about having hostname as a config. Probably be good to get other opinions on it.

Reply:
Looking back, there is no real need to update the config. This value can be accessed and added to the commonAttributes from the Agent, without changing the config at all.